lib/configuration/configuration_parser.js

```diff
@@ -0,0 +1,50 @@
+const Logger = require('../logger');
+const fsPromises = require('fs/promises');
+const clientConfiguration = require('./client_configuration');
+
+exports.parse = async function (configFilePath) {
+  const configFile = await tryToFind(configFilePath);
+  return configFile == null ? null : tryToParse(configFile);
+};
+
+function tryToFind (filePath) {
+  if (!filePath) {
+    return Promise.resolve(null);
+  }
+  return fsPromises.readFile(filePath, { encoding: 'utf8' })
+    .catch(err => {
+      const errorMessage = 'Finding easy logging configuration failed';
+      Logger.getInstance().error(errorMessage + '\n' + err.toString());
```

Comment:
do we need to log here? I would move logging one level higher and log only in the single exported from this file function

Reply:
removed